detection_rules/main.py

```diff
@@ -127,7 +127,7 @@ def import_rules_into_repo(input_file: click.Path, required_only: bool, action_c
                            exceptions_import: bool, directory: click.Path, save_directory: click.Path,
                            action_connectors_directory: click.Path, exceptions_directory: click.Path,
                            skip_errors: bool, default_author: str, strip_none_values: bool):
-    """Import rules from json, toml, yaml, or Kibana exported rule file(s)."""
+    """Import rules from json, toml, or yaml files containing Kibana exported rule(s)."""
```

Comment:
This is the key distinction here. Not any detection rule toml file. It should be Kibana exported content.